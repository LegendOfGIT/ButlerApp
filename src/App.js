import React, { Component } from 'react';
import logo from './logo.svg';
import './App.css';
import './Components/Channel.css';
import './Components/ChannelItemComponent.css';
import 'bootstrap/dist/css/bootstrap.css';
import 'font-awesome/css/font-awesome.css'
import Channel from './Components/ChannelComponent.jsx';

class App extends Component {
  render() {
      return (
          <div className="App">
            <div className="App-header">
              <img src={logo} className="App-logo" alt="logo" />
              <h2>Welcome to React</h2>
            </div>
            <p className="App-intro">
              To get started, edit <code>src/App.js</code> and save to reload.
            </p>

            <Channel />
          </div>
      );
  }
}

export default App;
