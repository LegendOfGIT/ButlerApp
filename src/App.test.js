import React from 'react';
import TestRenderer from 'react-test-renderer';
import App from './App';

it('renders without crashing', () => {
    expect(TestRenderer.create(<App/>).toJSON()).toMatchSnapshot();
});
