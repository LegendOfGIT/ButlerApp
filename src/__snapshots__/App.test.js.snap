// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="App"
>
  <div
    className="App-header"
  >
    <img
      alt="logo"
      className="App-logo"
      src="logo.svg"
    />
    <h2>
      Welcome to React
    </h2>
  </div>
  <p
    className="App-intro"
  >
    To get started, edit 
    <code>
      src/App.js
    </code>
     and save to reload.
  </p>
  <div
    className="Channel"
  >
    <h2>
      Test-Stream
    </h2>
    <div
      className="ChannelItem"
    >
      <div>
        Title-Image
      </div>
      <div>
        Title
      </div>
      <div>
        Information-Header
      </div>
      <div>
        Description
      </div>
    </div>
    <div
      className="ChannelItem"
    >
      <div>
        Title-Image
      </div>
      <div>
        Title
      </div>
      <div>
        Information-Header
      </div>
      <div>
        Description
      </div>
    </div>
    <div
      className="ChannelItem"
    >
      <div>
        Title-Image
      </div>
      <div>
        Title
      </div>
      <div>
        Information-Header
      </div>
      <div>
        Description
      </div>
    </div>
  </div>
</div>
`;
